packages/react/src/components/hero/Hero.stories.tsx

```diff
@@ -0,0 +1,546 @@
+  };
+  return (
+    <Hero {...heroProps}>
+      <DefaultContent />
+    </Hero>
+  );
+};
+
+WithoutImageAndKoros.argTypes = {
+  ...getDisabledControl('variant'),
+  ...getDisabledControl('imageSrc'),
+  ...getDisabledControl('koros'),
+};
+
+export const BackgroundImage = (args) => {
+  const heroProps: HeroProps = {
+    theme: { '--background-color': '#fff', ...args.theme },
+    koros: { ...args.koros },
+    imageSrc: args.imageSrc || imageFile,
+    variant: 'backgroundImage',
+  };
+  return (
+    <Hero {...heroProps}>
+      <DefaultContent buttonTheme="black" />
+    </Hero>
+  );
+};
+
+BackgroundImage.argTypes = {
+  ...getDisabledControl('centeredContent', true),
+  ...getDisabledControl('variant'),
+};
+
+export const DiagonalKoros = (args) => {
+  const heroProps: HeroProps = {
+    theme: { '--background-color': '#f5a3c7', '--color': '#000', ...args.theme },
+    koros: { ...args.koros },
+    imageSrc: args.imageSrc || imageFile,
+    variant: 'diagonalKoros',
+  };
+
+  return (
+    <Hero {...heroProps}>
+      <DefaultContent buttonTheme="black" />
+    </Hero>
+  );
+};
+
+DiagonalKoros.argTypes = {
+  ...getDisabledControl('centeredContent', true),
+  ...getDisabledControl('variant'),
+};
+
+export const ImageBottom = (args) => {
+  const heroProps: HeroProps = {
+    koros: { ...args.koros },
+    centeredContent: args.centeredContent,
+    theme: { '--background-color': '#fff', '--image-position': 'bottom left', ...args.theme },
+    imageSrc: args.imageSrc || imageFile,
+    variant: 'imageBottom',
+  };
+  return (
+    <Hero {...heroProps}>
+      <DefaultContent />
+    </Hero>
+  );
+};
+ImageBottom.storyName = 'Bottom image';
+ImageBottom.argTypes = {
+  ...getDisabledControl('variant'),
+};
+
+export const PlaygroundForKoros = (args) => {
+  const heroProps: HeroProps = {
+    koros: {
+      type: args.type,
+      dense: !!args.dense,
+      hide: !!args.hide,
+      ...args.koros,
+    },
+    theme: {
+      '--background-color': '#9fc9eb',
+      '--koros-color': args.color || '#9fc9eb',
+      '--diagonal-koros-position': args.diagonalKorosPosition,
+    },
+    imageSrc: imageFile,
+    variant: args.variant,
+  };
+
+  return (
+    <Hero {...heroProps}>
+      <DefaultContent />
+    </Hero>
+  );
+};
+
+PlaygroundForKoros.argTypes = {
+  ...getDisabledControl('centeredContent'),
+  ...getDisabledControl('imageSrc'),
+  ...getDisabledControl('theme'),
+  type: {
+    defaultValue: 'basic',
+    description: 'Koros type',
+    control: {
+      type: 'select',
+      options: ['basic', 'beat', 'pulse', 'storm', 'wave', 'calm'],
+    },
+  },
+  color: {
+    control: { type: 'color' },
+    description: 'Koros color. Default is "--background-color"',
+  },
+  hide: {
+    control: 'boolean',
+    description: 'Hide koros. Most variants override this setting.',
+  },
+  dense: {
+    control: 'boolean',
+    description: 'Use dense koros version or not',
+  },
+  diagonalKorosPosition: korosPosition,
```

Comment:
Hide, dense and flipHorizontal are Koros props but I don't know if that needs to mentioned. Maybe add the notion about this `diagonalKorosPosition` not being an actual component prop?

Reply:
I'll mention it is set via theme.

Reply:
New descriptions and summaries have been set.